src/style.css

```diff
@@ -0,0 +1,82 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.page {
+  font-family: Avenir, serif;
+}
+
+.page__content {
+  margin: 0;
+}
+
+.search__input {
+  display: flex;
+  box-sizing: border-box;
```

Comment:
Got your other suggestions, tysm, also added variable for colour hex 'cause it is used more than once. But wanted to ask about this one. You mean just 25 line or 24 too? Because moving 24 crashes all styling. Do I need to restyle it that way?